### ISSUE TITLE
RS1026 false positive for overridden initialize method

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn-analyzers#4625]
RS1026 should not report a diagnostic on the following code:

```csharp
public abstract class BaseAnalyzer : DiagnosticAnalyzer
{
    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => throw null;

    public override void Initialize(AnalysisContext context)
    {
        context.EnableConcurrentExecution();
        context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
    }
}

public class DerivedAnalyzer : BaseAnalyzer
{
    public override void Initialize(AnalysisContext context)
    {
        base.Initialize(context);
    }
}
```

**Comment by mattleibow** [comment]
This is especially annoying with larger API sets with overrides to extend/change behavior. Overrides are not breaking so maybe can have a flag to only list APIs that are actually changing API/ABI?
